Makefile

```diff
@@ -240,7 +240,7 @@ endif
 
 # On Windows only build tsh. On all other platforms build teleport, tctl,
 # and tsh.
-BINS_default = teleport tctl tsh tbot fdpass-teleport
+BINS_default = teleport tctl tsh tbot fdpass-teleport teleport-update
```

Comment:
Where is the actual target for this?

Reply:
https://github.com/gravitational/teleport/blob/c7d8791c68423c57f36d40d0a7e28720a1ead94e/Makefile#L398-L400